apiserver/facades/client/usermanager/usermanager_test.go

```diff
@@ -821,3 +821,88 @@ func (s *userManagerSuite) TestRemoveUserBulkSharedModels(c *gc.C) {
 	c.Assert(alice.IsDeleted(), jc.IsTrue)
 
 }
+
+func (s *userManagerSuite) TestResetPassword(c *gc.C) {
```

Comment:
I can certainly add test for (a) although I think it is a redundant test.
test for (b) is definitely a State level since there is no logic here that even queries whether user has a password or not. And that functionality is indeed tested in state tests added in earlier PR.